### PR TITLE
Rebase

### PR DESCRIPTION
**Pull request by jsirianni**
- added tcp plugin
- added udp plugin
- updated oracle plugin
- updated ubiquiti plugin
- updated haproxy plugin
- removed CI as it will not pass until Stanza is running open telemetry directly, until then we will just disable it.

### CODE REVIEW

File: plugins/ubiquiti.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ pipeline:
     attributes:
       log_type: ubiquiti
       plugin_id: {{ .id }}
-    add_labels: true
+    add_attributes: true
```
Comment (schmikei) [inline plugins/ubiquiti.yaml:47]: good catch

File: plugins/haproxy.yaml [hunk 2]
```diff
@@ -65,14 +65,14 @@ pipeline:
         expr: '$body.message != nil and $body.message matches "^(\\s)?[^:]+:[^\\s]+\\s+\\[[^\\]]+\\]\\s+[^\\s]+\\s+[^/]+/[^\\s]+\\s+[^/]+/[^/]+/[^\\s]+\\s+[^\\s]+\\s+[\\w-]{2}\\s+[^/]+/[^/]+/[^/]+/[^/]+/[^\\s]+\\s+[^/]+/[\\d]+"'
       - output: error_parser
         expr: '$body.message != nil and $body.message matches "^\\s*\\[[^\\]]+\\]\\s*.*"'
-        attributes:
+        labels:
```
Comment (schmikei) [inline plugins/haproxy.yaml:68]:
```suggestion
        attributes:
```

missed one

File: plugins/haproxy.yaml [hunk 3]
```diff
@@ -112,7 +112,7 @@ pipeline:
         expr: '$body.message != nil and $body.message matches "^{.*}$|^{.*}\\n$"'
       - output: error_parser
         expr: '$body.message != nil and $body.message matches "^\\s*\\[[^\\]]+\\]\\s*.*"'
-        attributes:
+        labels:
```
Comment (schmikei) [inline plugins/haproxy.yaml:115]: missed here as well

File: plugins/haproxy.yaml [hunk 4]
```diff
@@ -178,8 +178,8 @@ pipeline:
     to: '$attributes["frontend_ssl_ciphers"]'
   - id: method_move
     type: move
-    if: '$attributes.method != nil'
```
Comment (schmikei) [inline plugins/haproxy.yaml:181]: i don't think this is right?

**Review by schmikei — CHANGES_REQUESTED** [review]
just some potential conflicts with haproxy

File: plugins/haproxy.yaml (hunk 4 above)
Comment (jsirianni) [reply plugins/haproxy.yaml:181]: Im not sure what that is, this is how it looks
```yaml
  - id: method_move
    type: move
    if: '$body.method != nil'
    from: '$body.method'
    to: '$attributes["method"]'
```

File: plugins/haproxy.yaml [hunk 5]
```diff
@@ -165,7 +165,7 @@ pipeline:
 # protocol
 # protocol_version
 # referer
-  # Promote fields to attributes
+  # Promote fields to labels
```
Comment (schmikei) [inline plugins/haproxy.yaml:168]:
```suggestion
  # Promote fields to attributes
```

**Review by schmikei — APPROVED** [review]

**Pull request merged**